src/swsssdk/dbconnector.py

```diff
@@ -1,20 +1,156 @@
 """
 Database connection module for SwSS
 """
-from . import _connector_map, logger
+from . import logger
 from .interface import DBInterface
-
+import os
+import json
 
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
+class SonicDBConfig(object):
+    SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_init = False
+    _sonic_db_config = {}
+
+    @staticmethod
+    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
+        """
+        Get multiple database config from the database_config.json
+        """
+        if SonicDBConfig._sonic_db_config_init == True:
+            return
+
+        try:
+            if os.path.isfile(sonic_db_file_path) == False:
+                sonic_db_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config', 'database_config.json')
```

Comment:
>sonic_db_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config', 'database_config.json') [](start = 16, length = 111)

We don't want to see this case in production. Can we add a syslog warning?

Reply:
added warning log